### PR TITLE
Added CRUD operation for Flight

### PR DESCRIPTION
**Pull request by SaniyaKumawat123**
Added CRUD operation for Flight:-
  1. Add Flight
  2. get all flights
  3. Update Flight
  4. Delete flight

### CODE REVIEW

File: src/main/java/com/tripply/booking/entity/Flight.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+@Data
+@Entity
+@Table(name = "flights")
+public class Flight {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long flightId;
+
+    @Column(nullable = false, unique = true)
```
Comment (hitsa29) [inline src/main/java/com/tripply/booking/entity/Flight.java:18]: what is airline and why its unique ?

File: src/main/java/com/tripply/booking/service/FlightService.java [hunk 2]
```diff
@@ -4,8 +4,13 @@
 import com.tripply.booking.model.request.FlightRequest;
 import com.tripply.booking.model.response.FlightResponse;
 
+import java.util.List;
+
 public interface FlightService {
 
 	ResponseModel<FlightResponse> addflight(FlightRequest flightRequest);
+	ResponseModel<List<FlightResponse>> getAllflights();
```
Comment (hitsa29) [inline src/main/java/com/tripply/booking/service/FlightService.java:12]: don't return responseModel from service. return simply FlightResponse or List<FlightResponse> wherever is required.

Comment (SaniyaKumawat123) [reply src/main/java/com/tripply/booking/service/FlightService.java:12]: We are using the ResponseModel to maintain a standardized response format across all services.

File: src/main/java/com/tripply/booking/entity/Flight.java (hunk 1 above)
Comment (SaniyaKumawat123) [reply src/main/java/com/tripply/booking/entity/Flight.java:18]: I have added a new variable called flightNo to represent the flight number and made it unique. and airline represents the name of the flight.

**Review by Jatin-Ongraph — APPROVED** [review]

**Pull request merged**